model/README.MD

```diff
@@ -0,0 +1,24 @@
+# Spine model verification tools
+
+This directory contains modules that implement compile time verification of the Spine model.
+
+To enable the verification in a project, apply the following Gradle config to subprojects that may 
+contain Spine model elements:
+ ```groovy
+ buildscript {
+     dependencies {
+         classpath "io.spine.tools:spine-model-verifier:${spineVersion}"
+     }
+ }
+ 
+ apply plugin: 'io.spine.tools.spine-model-verifier'
+ 
+ compileJava {
+     options.compilerArgs += ["-processor", "io.spine.model.assemble.AssignLookup", "-AspineDirRoot=${rootDir}"]
```

Comment:
`-AspineDirRoot` seems to be a typo. Isn't it `-D` that preceeds the command-line parameters addressed to the application?